initScripts/x86_64/WindowsServer_2016/Docker_17.06.ps1

```diff
@@ -51,6 +54,14 @@ Function install_prereqs() {
   & "$NODE_SHIPCTL_LOCATION/$NODE_ARCHITECTURE/$NODE_OPERATING_SYSTEM/install.ps1"
 }
 
+Function add_firewall_rule() {
+  $existingFWRule = Get-NetFirewallRule -DisplayName shippable-docker -ErrorAction SilentlyContinue
+
+  if (!($ExistingFWRule)) {
```

Comment:
Maybe it's better to Remove-NetFirewallRule and re-create it, so we always know it's good to go after running the script.